pkg/fleet/installer/repository/repository.go

```diff
@@ -93,9 +93,14 @@ func (r *Repository) GetState() (State, error) {
 	if err != nil {
 		return State{}, err
 	}
+	stable := repository.stable.Target()
+	experiment := repository.experiment.Target()
+	if experiment == stable {
+		experiment = ""
+	}
 	return State{
-		Stable:     repository.stable.Target(),
-		Experiment: repository.experiment.Target(),
+		Stable:     stable,
+		Experiment: experiment,
```

Comment:
I get why we're doing this, but I'm afraid we'd miss cases where the experiment isn't there at all. Maybe we should add a third field?